tests/clients/test_tournaments.py

```diff
@@ -17,3 +19,9 @@ def test_swiss_result(self):
     def test_arenas_result(self):
         res = list(Client().tournaments.stream_results("hallow23", limit=3))
         validate(List[ArenaResult], res)
+
+    @skip_if_older_3_dot_10
+    @pytest.mark.vcr
+    def test_team_standings(self):
+        res = list(Client().tournaments.get_team_standings("fake"))
```

Comment:
The issue lies here, you will not be able to get standings if you do not provide a correct tournament id. It'll fix it.